### DISCUSSION TITLE
Suggestion: add hacktoberfest topic to the project

### DISCUSSION THREAD

**Discussion by krnlde** [react-hook-form/react-hook-form#6822] [General]
Hey Bill,
For anyone participating in the this year's [hacktoberfest](http://hacktoberfest.digitalocean.com/) (like me) it would be awesome if you added the `hacktoberfest` topic to the repository. Alternatively you could add the `hacktoberfest-approved` label to all issues and PRs you'd like to be counted.

You can read more on their website or ask me if you have any questions :)

Cheers mate!
Kai

**Comment by bluebill1049** [comment]
awesome! will do now.

**Comment by bluebill1049** [comment]
<img width="1332" alt="Screen Shot 2021-10-16 at 8 43 13 am" src="https://user-images.githubusercontent.com/10513364/137556727-d9573099-2a3f-4ae5-a90a-d3d956f7fbfb.png">

label added 👍

**Reply by krnlde** [reply]
Hey Bill :) almost, you'd need to append the label to every pull request or issue by hand. The other option would be to add `hacktoberfest` to the topics of the repo (in the about section). See https://github.com/reactstrap/reactstrap for example.

Cheers,
Kai

**Reply by bluebill1049** [reply]
thanks! done.
